pkg/gwtestctl/client.go

```diff
@@ -130,12 +131,28 @@ func getClient(target string) (dataplane.ConfigServiceClient, func() error, erro
 		return nil, nil, fmt.Errorf("parsing target: %w", err)
 	}
 
+	var conn *grpc.ClientConn
+
 	if scheme == unixScheme {
-		target = fmt.Sprintf("unix-abstract:%s", target)
+		if target == "" {
+			return nil, nil, fmt.Errorf("unix socket target cannot be empty")
+		}
+		if target[0] != '/' {
+			target = "/" + target
+		}
```

Comment:
What does this do?

Wouldn't it be more robust to use `path.join()` to modify system paths?

Reply:
Yeah, it's just not needed, if an incorrect path is passed - so be it